runtime/router/router.go

```diff
@@ -30,9 +30,10 @@ import (
 // Router dispatches http requests to a registered http.Handler.
 // It implements a similar interface to the one in github.com/julienschmidt/httprouter,
 // the main differences are:
-// 1. this router does not treat "/a/:b" and "/a/b/c" as conflicts (https://github.com/julienschmidt/httprouter/issues/175)
+// 1. this router does not treat "/a/:b" and "/a/b/c" as conflicts (https://github.com/julienschmidt/httprouter/issues/175) except for whitelisted paths
```

Comment:
comment update makes it read as though if they are not whitelisted, they are conflicting which isn't true.

Reply:
this is not addressed yet.